tests/unit/plugins/module_utils/test_system_access_users_utils.py

```diff
@@ -731,41 +880,35 @@ def test_password_verify_returns_true_on_match(mock_run_command: MagicMock):
     "ansible_collections.puzzle.opnsense.plugins.module_utils.opnsense_utils.run_command"
 )
 def test_password_verify_returns_false_on_difference(mock_run_command: MagicMock):
-
     # Mock the return value of the run_command to simulate a password match
     mock_run_command.return_value = {
         "stdout": "1",
         "stderr": None,
     }
-
     # Call the function with test data
-    test_password_matches = password_verify(
-        password="test_password_1",
-        existing_user_password="$2y$11$pSYTZcD0o23JSfksEe1231345h9vp7OOQN.v35E1rag49cEc6",
+    test_password_matches = hash_verify(
+        plain_string="test_password_1",
+        existing_hashed_string="$2y$11$pSYTZcD0o23JSfksEe1231345h9vp7OOQN.v35E1rag49cEc6",
     )
-
     # Assert that the function returns True for a password match
     assert not test_password_matches
 
 
 @patch(
     "ansible_collections.puzzle.opnsense.plugins.module_utils.opnsense_utils.run_command"
 )
-def test_password_verify_returns_OPNSensePasswordVerifyReturnError(
+def test_password_verify_returns_OPNSenseHashVerifyReturnError(
     mock_run_command: MagicMock,
 ):
-
     # Mock the return value of the run_command to simulate a password match
     mock_run_command.return_value = {
         "stdout": None,
         "stderr": "this an error",
     }
-
-    with pytest.raises(OPNSensePasswordVerifyReturnError) as excinfo:
+    with pytest.raises(OPNSenseHashVerifyReturnError) as excinfo:
```

Comment:
```suggestion
    with pytest.raises(OPNsenseHashVerifyReturnError) as excinfo:
```

Reply:
Thanks for the suggestion, fixed in c7e6fb8344cd60522a5c1fbdc273923e17d87cd7.